network-api/networkapi/wagtailpages/factory/youtube_regrets_page.py

```diff
@@ -30,6 +34,25 @@ class Meta:
     regret_stories = Faker("streamfield", fields=["regret_story"] * 28)
 
 
+class AppInstallPageFactory(PageFactory):
+    class Meta:
+        model = AppInstallPage
+        exclude = (
+            "title_text",
+            "header_text",
+            "header",
+        )
+
+    title = "Regrets Reporter Page"
+    slug = "regretsreporter"
+    hero_heading = Faker("text", max_nb_chars=50)
+    hero_subheading = Faker("text", max_nb_chars=50)
+    hero_background = SubFactory(image_factory.ImageFactory)
+    hero_video = Faker("url")
```

Comment:
```suggestion
```

Since we need a specific format for [embedding YouTube or Vimeo videos](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/52a2fdb2bcf50ac4203d6fa17774100e4d83fbe2/network-api/networkapi/wagtailpages/pagemodels/app_install.py#L32-L33) (like youtube.com/embed/xyz...), I think we can remove this field from the factory, since its optional anyways